Add media player entities section to Shelly documentation (#44969)
home-assistant/home-assistant.github.io: source/_integrations/shelly.markdown

For generation 2+ hardware, it's possible to select if a device's input is connected to a button or a switch. Binary sensors are created only if the **Input Mode** is set to `Switch`. When the **Input Mode** is set to `Button` you need to use events for your automations.

## Media player entities

Wall Display devices with firmware 2.2 or newer can function as media players. The integration creates media player entities for them.

> The Wall Display media player can play local audio files (you need to upload them to the Wall Display media library) and internet radio stations that you have added to your favorites in the device interface.

Reviewer: ```suggestion
The Wall Display media player can play the following audio formats:

- Local audio files uploaded to the Wall Display media library
- Internet radio stations added to your favorites
```
It's a bit of a long sentence so I took a stab at breaking it up. Feel free to ignore if you don't like the proposal.
